entitlement/src/main/java/org/killbill/billing/entitlement/api/DefaultEntitlementContext.java

```diff
@@ -117,6 +119,16 @@ public DefaultEntitlementContext(final OperationType operationType,
         this.tenantId = tenantId;
     }
 
+    private static <T> Iterable<T> merge(final Iterable<T> prevValues, final Iterable<T> newValues) {
```

Comment:
I would expect a `merge` method to merge the two input into one - but looks like instead we either return one or the other ? Isn't more like a `lastNonEmptyOrNull` kind of thing?

Reply:
Indeed - fixed in 30e911ba0ac468d5d9356ce9a5da270d4cadfc9e.